File: dev-docs/snippets/collection-type-configs/list.ts
import type { JsBaoClient } from "js-bao-wss-client";

// List every collection type configuration for the current app.
export async function list(client: JsBaoClient) {
  // #region example
  const configs = await client.collectionTypeConfigs.list();
  // #endregion example
  return configs;
}

File: dev-docs/snippets/collection-type-configs/list.swift
import JsBaoClient

// List every collection type configuration for the current app. Returns a
// typed `[CollectionTypeConfigInfo]`.
func list(client: JsBaoClient) async throws {
  // #region example
  let configs = try await client.collectionTypeConfigs.list()
  // #endregion example
  _ = configs
}
